server/src/domain/library/library.service.ts

```diff
@@ -305,9 +304,8 @@ export class LibraryService extends EventEmitter {
     this.logger.verbose(`Queuing refresh of ${assetPaths.length} asset(s)`);
 
     // We perform this in batches to save on memory when performing large refreshes (greater than 1M assets)
-    const batchSize = 5000;
-    for (let i = 0; i < assetPaths.length; i += batchSize) {
-      const batch = assetPaths.slice(i, i + batchSize);
+    for (let i = 0; i < assetPaths.length; i += LIBRARY_SCAN_BATCH_SIZE) {
+      const batch = assetPaths.slice(i, i + LIBRARY_SCAN_BATCH_SIZE);
```

Comment:
The call to `scanAssets` is already batched now so this is redundant.

Reply:
Fixed